schemas/racksdb.yml

```diff
@@ -322,6 +322,14 @@ _objects:
         type: list[str]
         description: The list of tags associated to this datacenter.
         optional: true
+      longitude:
```

Comment:
This, one could define the longitude without the latitude and vice-versa. It would be better with a new object, attached to coordinates or position attribute, in which both longitude and latitude are required.